### PR TITLE
[v6r13] Minor fixes

### PR DESCRIPTION
**Pull request by atsareg**

### CODE REVIEW

File: ConfigurationSystem/scripts/dirac-admin-add-resources.py [hunk 1]
```diff
@@ -313,10 +313,10 @@ def checkUnusedSEs():
         changeSet.add( ( seSection, 'VO', ','.join( seVOs ) ) )
         accessSection = cfgPath( seSection, 'AccessProtocol.1' )
         changeSet.add( ( accessSection, 'Protocol', 'srm' ) )
-        changeSet.add( ( accessSection, 'ProtocolName', 'SRM2' ) )
+        changeSet.add( ( accessSection, 'PluginName', 'SRM2' ) )
```
Comment (fstagni) [inline ConfigurationSystem/scripts/dirac-admin-add-resources.py]: Why is this necessary? Does it imply a change in the CS?

Comment (atsareg) [inline ConfigurationSystem/scripts/dirac-admin-add-resources.py]: This is a new non-contradictory option name to be used now in the CS. The backward compatibility with the old ambiguous  ProtocolName is preserved.

**Comment by chaen** [comment]
And we said no PR called "minor fixes" anymore ;-)
